File: pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

#define PYRAMIDHEIGHT 9

int main(void)
{
    int height;

    while (true)
    {
        height = get_int("Height: ");

        if (0 < height && height < PYRAMIDHEIGHT)
        {
            break;
        }
        continue;
    }

    for (int row = 0; row < height; row++)
    {
        for (int cursorplacement = 0; cursorplacement < height; cursorplacement++)
        {
            if (row + cursorplacement < height - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("  ");
        for (int cursorplacementright = 0; cursorplacementright < row + 1; cursorplacementright++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/sentimental-mario-more/mario.py
PYRAMIDHEIGHT = 9

while True:
    try:
        height = int(input("Height: ").strip())

        if 0 < height and height < PYRAMIDHEIGHT:
            break
        continue
    except ValueError:
        continue

row = 0
while row < height:

    cursorplacement = 0
    while cursorplacement < height:

        if row + cursorplacement < height - 1:
            print(" ", end=(""))
        else:
            print("#", end=(""))

        cursorplacement += 1

    print("  ", end=(""))

    cursorplacementright = 0
    while cursorplacementright < (row + 1):

        print("#", end=(""))
        cursorplacementright += 1

    print()
    row += 1
